src/lib/index.ts

```diff
@@ -421,6 +420,37 @@ export class SourceFileConverter {
               );
               partTypes.push('attribute');
               continue;
+            } else if (name.startsWith('@')) {
+              // Convert event binding into a jsaction.
```

Comment:
Let's pull this out into a separate function now, even if it's in the same module. Just to reduce the size of this function.